src/ui/Fields/CommentField.tsx

```diff
@@ -1,19 +1,22 @@
 import Stack from '@mui/material/Stack';
 import { FilledInput } from '@mui/material';
 import { Typography } from '../Typography';
-import React from 'react';
 import D from 'i18n';
 
+interface CommentFieldProps {
+  value: string;
+  onChange: (v: string) => void;
+}
 
 /**
  * @param {string} value
  * @param {(v: string) => void} onChange
  * @param {import('@mui/material').FilledInputProps} props
  * @constructor
  */
-export function CommentField({ value, onChange, ...props }) {
+export function CommentField({ value, onChange, ...props }: CommentFieldProps) {
   const maxChar = 999;
-  const handleChange = e => {
+  const handleChange = (e: { target: { value: string } }) => {
```

Comment:
Why not using the out-of-the-box `ChangeEventHandler<HTMLInputElement | HTMLTextAreaElement>` ?